src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/zk/ZooKeeperBolt.java

```diff
@@ -72,14 +69,10 @@ protected void init() {
     }
 
     private void initZk() {
```

Comment:
Do you really need both `init()` and `initZk()`?

Reply:
actually not. We can move initZk to init